components/scim/org.wso2.carbon.identity.scim.common/src/main/java/org/wso2/carbon/identity/scim/common/config/SCIMProviderDAO.java

```diff
@@ -72,10 +73,9 @@
             log.error(errorMsg, e);
             throw new IdentitySCIMException(errorMsg, e);
         } catch (SQLException e) {
-            log.error("Error when executing the SQL : " + SQLQueries.GET_ALL_PROVIDERS_SQL);
-            log.error(e.getMessage(), e);
+            log.error("Error when executing the SQL : " + SQLQueries.GET_ALL_PROVIDERS_SQL, e);
             throw new IdentitySCIMException("Error when reading the SCIM Providers information from " +
```

Comment:
when we log the exception with stack trace as an error log, no need to throw with stack trace. fix like this

 log.error("Error when executing the SQL : " + SQLQueries.GET_PROVIDER_SQL, e);
             throw new IdentitySCIMException("Error when reading the SCIM Provider information from " +
                  "the persistence store.");